nodeup/pkg/model/containerd.go

```diff
@@ -220,11 +226,19 @@ func (b *ContainerdBuilder) Build(c *fi.ModelBuilderContext) error {
 			var packageTask fi.Task
 			if dv.PlainBinary {
 				packageTask = &nodetasks.Archive{
-					Name:            "containerd",
-					Source:          dv.Source,
-					Hash:            dv.Hash,
-					TargetDir:       "/usr/bin/",
-					StripComponents: 1,
+					Name:      "containerd.io",
+					Source:    dv.Source,
+					Hash:      dv.Hash,
+					TargetDir: "/",
+					MapFiles: map[string]string{
+						"./usr/local/bin/ctr":                     "/usr/bin",
```

Comment:
I'm wondering if we should map directories, as this likely makes it harder to support new tar.gz bundles we don't yet know about (they may have added a file)...

Reply:
**Very** good point. Updated and works perfectly.